apps/spruce/src/components/AprilFoolsEvergreenRedesign/index.tsx

```diff
@@ -0,0 +1,155 @@
+  (_, ref) => {
+    const [modalOpen, setModalOpen] = useState(
+      Cookies.get(SEEN_COMIC_SANS_PRANK) !== "true",
+    );
+    const [comicSansEnabled, setComicSansEnabled] = useState(false);
+    const [isFollowUp, setIsFollowUp] = useState(false);
+    const { sendEvent } = useNavbarAnalytics();
+    useImperativeHandle(ref, () => ({
+      openModal: () => {
+        sendEvent({ name: "Clicked enable april fools link in navbar" });
+        setModalOpen(true);
+        setIsFollowUp(false);
+      },
+    }));
+
+    const triggerComicSansWithAnimation = () => {
+      setModalOpen(false); // Hide original modal
+
+      setComicSansEnabled(true);
+
+      // Now start the 10s timer for the follow-up prank modal
+      setTimeout(() => {
+        setIsFollowUp(true);
+        setModalOpen(true);
+        Cookies.set(SEEN_COMIC_SANS_PRANK, "true", { expires: 2 });
+      }, 8000);
+    };
+
+    useEffect(() => {
+      if (!comicSansEnabled) return;
+
+      const trailElements: HTMLSpanElement[] = [];
+
+      const handleMouseMove = (e: MouseEvent) => {
+        const trail = document.createElement("span");
+        trail.textContent = "✨";
+        trail.style.position = "fixed";
+        trail.style.left = `${e.clientX}px`;
+        trail.style.top = `${e.clientY}px`;
+        trail.style.pointerEvents = "none";
+        trail.style.zIndex = "2147483647";
+        trail.style.transform = "translate(-50%, -50%)";
+        trail.style.fontSize = "20px";
+        trail.style.opacity = "1";
+        trail.style.transition =
+          "opacity 0.6s ease-out, transform 0.6s ease-out";
+        document.body.appendChild(trail);
+        trailElements.push(trail);
+
+        requestAnimationFrame(() => {
+          trail.style.opacity = "0";
+          trail.style.transform = "translate(-50%, -50%) scale(2)";
+        });
+
+        setTimeout(() => {
+          trail.remove();
+        }, 600);
+      };
+
+      window.addEventListener("mousemove", handleMouseMove);
+
+      return () => {
+        window.removeEventListener("mousemove", handleMouseMove);
+        trailElements.forEach((el) => el.remove());
+      };
+    }, [comicSansEnabled]);
+
+    return (
+      <>
+        {comicSansEnabled && (
+          <Global
+            styles={css`
+              * {
+                font-family: "Comic Sans MS", "Comic Sans", cursive !important;
+                letter-spacing: 0.1em;
+                text-transform: uppercase;
+                cursor:
+                  url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg'  width='40' height='48' viewport='0 0 100 100' style='fill:black;font-size:24px;'><text y='50%'>🦄</text></svg>")
+                    16 0,
+                  auto;
+              }
+            `}
+          />
+        )}
+
+        <MarketingModal
+          buttonText={
+            isFollowUp
+              ? "The Evergreen team rocks for this!"
```

Comment:
My only problem with this copy is that it's not 100% clear that clicking this button will cause the Comic Sans state to persist

Reply:
I think the text above "Comic Sans is here to stay" indicates that happy to swap it if you have any suggestions

Reply:
something like `"Continue the experience"` could be appropriate